src/test/prob/exponential/exponential_test.hpp

```diff
@@ -28,6 +28,8 @@ class AgradDistributionsExponential : public AgradDistributionTest {
   void invalid_values(vector<size_t>& index, 
           vector<double>& value) {
```

Comment:
y = 0 is a valid value, and in the testing framework, our finite diff tests will fail if we include y = 0 in the valid values. How should we get around this?